history/titles/00_k_black_morass.txt

```diff
@@ -38,7 +38,8 @@ c_rockard = {
 	}
 	583.1.1={
 		liege=k_blackrock_clan
-		holder=310050 #Kanath[patron]
+		#holder=310050 #Kanath[patron] will become landed again once the map gets bigger
+		holder=thok
```

Comment:
If you remove patron character from bookmarks completely
give him land elsewhere or dont add historical one yet